emrun

```diff
@@ -1441,8 +1445,8 @@ def main():
       browser_stderr_handle = open(options.log_stderr, 'ab')
 
   if not options.no_server:
-    logv('Starting web server in port ' + str(options.port))
-    httpd = HTTPWebServer(('', options.port), HTTPHandler)
+    logv('Starting web server: http://%s:%i/' % (options.hostname, options.port))
+    httpd = HTTPWebServer((options.hostname, options.port), HTTPHandler)
```

Comment:
The `client_address` parameter for [`BaseHTTPServer.HTTPServer`](https://docs.python.org/2/library/basehttpserver.html#BaseHTTPServer.BaseHTTPRequestHandler)'s conveniently takes in a `host` as the first tuple element.